charts/osm/crds/remoteservice.yaml

```diff
@@ -0,0 +1,65 @@
+{{ if .Values.enableMulticluster }}
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: remoteservices.multicluster.openservicemesh.io
+spec:
+  group: multicluster.openservicemesh.io
```

Comment:
As discussed offline we want this to be a part of the `config.openservicemesh.io` that already exists